common/main/scn.xml

```diff
@@ -797,21 +797,21 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 							<pattern>↑↑↑</pattern>
 						</dateTimeFormat>
 						<dateTimeFormat type="atTime">
-							<pattern>{1} 'a l’uri' {0}</pattern>
+							<pattern>{1} 'a' 'l'’'uri' {0}</pattern>
```

Comment:
These changes involve something I'm not familiar with: insertion of apostrophe in dateTimeFormat patterns.

Compare:

https://st.unicode.org/cldr-apps/v#/scn/Gregorian/1eb22f6111d66f88

https://st.unicode.org/cldr-apps/v#/de/Gregorian/1eb22f6111d66f88

https://st.unicode.org/cldr-apps/v#/fr/Gregorian/1eb22f6111d66f88

Reply:
Literal text consisting of A-Za-z _must_ be enclosed in straight single quotes to distinguish it from date-time pattern characters. Other literal text _may_ also be enclosed in straight quotes but need not be. This is just CLDRMody being overly pedantic about removing things other than A-Za-z from the text enclosed in straight quotes. A little annoying but OK.